components/KiboPayment.vue

```diff
@@ -0,0 +1,201 @@
+                  <font-awesome-icon icon="question-circle" class="fa-icon" />
+                </SfIcon>
+              </span>
+            </SfButton>
+            <label for="cvv" class="sf-input__label">{{ $t("Security Code") }}</label>
+          </div>
+          <div class="sf-input__error-message"><div class="display-none"></div></div>
+        </div>
+        <SfCheckbox
+          v-model="creditCardFormData.card.isCardInfoSaved"
+          name="keepcard"
+          :label="$t('Save payment method')"
+          class="credit-card-form__checkbox"
+          @change="validateInput()"
+        />
+      </div>
+    </template>
+  </SfPayment>
+</template>
+<script lang="ts">
+import { defineComponent } from "@vue/composition-api"
+import { SfPayment, SfRadio, SfInput, SfCheckbox, SfIcon, SfButton } from "@storefront-ui/vue"
+import { ref } from "@nuxtjs/composition-api"
+import creditCardType from "credit-card-type"
+import { usePaymentTypes } from "@/composables"
+import { creditCardPaymentGetters } from "@/lib/getters"
+import { defaultPaymentDetails } from "@/composables/helpers"
+
+export default defineComponent({
+  name: "KiboPayment",
+  components: {
+    SfPayment,
+    SfRadio,
+    SfInput,
+    SfCheckbox,
+    SfIcon,
+    SfButton,
+  },
+  setup(_props, _context) {
+    const { loadPaymentTypes } = usePaymentTypes()
+    const paymentMethods = loadPaymentTypes()
+    const isCreditCardSelected = ref(false)
+    const creditCardFormData = ref(defaultPaymentDetails())
+
+    const selectedPaymentMethod = (fieldValue: string) => {
+      creditCardFormData.value.paymentType = fieldValue
+      isCreditCardSelected.value = fieldValue.toLowerCase() === "creditcard"
+    }
+
+    const validateInput = () => {
+      if (creditCardFormData.value.card.expiryDate) {
+        creditCardFormData.value.card.expireMonth = creditCardPaymentGetters.getExpireMonth(
+          creditCardFormData.value.card
+        )
+        creditCardFormData.value.card.expireYear = creditCardPaymentGetters.getExpireYear(
+          creditCardFormData.value.card
+        )
+      }
+
+      if (creditCardFormData.value.card.cardNumber) {
+        const ccardType = creditCardType(creditCardFormData.value.card.cardNumber)
+        creditCardFormData.value.card.cardType = ccardType.length
+          ? ccardType[0].type.toUpperCase()
+          : ""
+      }
+
+      _context.emit("input", {
+        ...creditCardFormData,
+      })
+    }
+
+    return {
+      paymentMethods,
+      selectedPaymentMethod,
+      creditCardFormData,
+      isCreditCardSelected,
+      validateInput,
+    }
+  },
+})
+</script>
+
+<style lang="scss">
+.sf-payment {
+  .sf-heading {
+    border-bottom-color: #f7f7f7;
+
+    &__title.h2 {
+      --heading-title-font-size: var(--font-size--xl);
+    }
+  }
+
+  .payment-methods {
+    background-color: #f7f7f7 !important;
+    border-bottom: none !important;
+    padding: var(--spacer-2xs) 0 !important;
```

Comment:
I have tried to avoid it but storefront CSS is overriding the component's CSS. I didn't find reason why its happing so use !important for there 3 properties. Other CSS are working as expected.